packages/grafana-toolkit/docker/grafana-plugin-ci/README.md

```diff
@@ -0,0 +1,52 @@
+# Using this docker image
```

Comment:
Maybe make it more clear that this docker image is intended for use in the plugin CI build systems.